Projects/hCoreUI/Sources/hForm/RadioFields/hRadioOptionField.swift

```diff
@@ -0,0 +1,91 @@
+import SwiftUI
+import hCore
+
+public struct hRadioOptionField: View {
+    private var placeholder: String
+    private var value: String
+    @State private var animate = false
+    private var useAnimation: Bool
+    @Binding var error: String?
+
+    @State var selected: String = ""
+
+    public var shouldMoveLabel: Binding<Bool> {
+        Binding(
+            get: { !value.isEmpty },
+            set: { _ in }
+        )
+    }
+
+    public init(
+        value: String,
+        placeholder: String? = nil,
+        error: Binding<String?>? = nil,
+        useAnimation: Bool = false
+    ) {
+        self.placeholder = placeholder ?? ""
+        self.value = value
+        self._error = error ?? Binding.constant(nil)
+        self.useAnimation = useAnimation
+    }
+
+    public var body: some View {
+        VStack(alignment: .leading, spacing: 0) {
+            hFieldLabel(
+                placeholder: placeholder,
+                animate: $animate,
+                error: $error,
+                shouldMoveLabel: shouldMoveLabel
+            )
+            .padding(.bottom, 13)
+
+            HStack(spacing: 16) {
+                getCheckBox(texts: [L10n.General.yes, L10n.General.no])
+                Spacer()
+            }
+        }
+        .padding(.vertical, 16)
+        .addFieldBackground(animate: $animate, error: $error)
+        .padding(.horizontal, 16)
```

Comment:
This component shouldn't take care of horizontal spacing. Components that wrap this should be in charge of this:
.padding(.horizontal, 16)

In our case hSection will cover this